python/cog/server/runner.py

```diff
@@ -237,7 +241,9 @@ def handle_error(task: RunnerTask) -> None:
                 if self._shutdown_event is not None:
                     self._shutdown_event.set()
             except BaseException:
```

Comment:
Again, I don't understand why we're catching `BaseException` and not re-raising it here. This will lead to some very confusing bugs.

Reply:
re-raised